Source/AMRInterpolator/SphericalGeometryUniform.hpp

```diff
@@ -0,0 +1,145 @@
+/* GRChombo
+ * Copyright 2012 The GRChombo collaboration.
+ * Please refer to LICENSE in GRChombo's root directory.
+ */
+
+#ifndef SPHERICALGEOMETRYUNIFORM_HPP_
+#define SPHERICALGEOMETRYUNIFORM_HPP_
+
+// Chombo includes
+#include "AlwaysInline.hpp"
+#include "MayDay.H"
+
+// Other includes
+#include "AlwaysInline.hpp"
+#include "IntegrationMethod.hpp"
+#include "MayDay.H"
+#include "SphericalGeometry.hpp"
+#include <array>
+#include <cmath>
+#include <string>
+
+// Chombo namespace
+#include "UsingNamespace.H"
+
+//! This SurfaceGeometry template class provides spherical shell geometry
+//! implementation for the SurfaceExtraction class
+//! u = u, v = phi
```

Comment:
`u = u` is not very helpful.
```suggestion
//! u = z/r, v = phi
```